.github/workflows/CI.yml

```diff
@@ -159,8 +159,8 @@ jobs:
         run: |
           npm install -g npm@8.19.3 &&  \
           npm ci && \
-          npm run plugin:install -- repository-name="githubator-plugin" repository-url="https://github.com/ditrit/githubator-plugin.git#0.3.0" && \
-          npm run plugin:install -- repository-name="terrator-plugin" repository-url="https://github.com/ditrit/terrator-plugin.git#0.7.1" && \
+          npm run plugin:install -- repository-name="githubator-plugin" repository-url="https://github.com/ditrit/githubator-plugin.git#0.3.1" && \
+          npm run plugin:install -- repository-name="terrator-plugin" repository-url="https://github.com/ditrit/terrator-plugin.git#0.8.0" && \
```

Comment:
maybe change commit name because its also updating the plugins version

Reply:
done